app/assets/stylesheets/styles/components/crops/components/tags-input.scss

```diff
@@ -0,0 +1,399 @@
+          }
+        }
+
+        .remove-button {
+
+          .input-group-lg {
+
+            .keywords {
+
+              .tags {
+
+                .tag-item {
+
+                  .remove-button {
+                    font-size: 20;
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+
+      .input {
+
+        .input-group-lg {
+
+          .keywords {
+
+            .tags {
+
+              .input {
+                height: 38px;
+                font-size: 18px;
+              }
+            }
+          }
+        }
+      }
+    }
+
+    &.ti-input-sm {
+
+      .input-group-sm {
+
+        .keywords {
+          min-height: 30;
+        }
+      }
+
+      .tags {
+
+        .input-group-sm {
+
+          .keywords {
+
+            .tags {
+              border-radius: 0;
+            }
+          }
+        }
+
+        .tag-item {
+
+          .input-group-sm {
+
+            .keywords {
+
+              .tags {
+
+                .tag-item {
+                  height: 22px;
+                  line-height: 21px;
+                  font-size: 12px;
+                  border-radius: 0;
+                }
+              }
+            }
+          }
+
+          .remove-button {
+
+            .input-group-sm {
+
+              .keywords {
+
+                .tags {
+
+                  .tag-item {
+
+                    .remove-item {
+
+                      font-size: 16px;
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+
+        .input {
+
+          .input-group-sm {
+
+            .keywords {
+
+              .tags {
+
+                .input {
+                  height: 22px;
+                  font-size: 12px;
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+.keywords {
+
+  .tags {
+
+    .tag-item {
+
+      &.selected {
+        background: -webkit-linear-gradient(top, #febbbb 0%, #fe9090 45%, #ff5c5c 100%);
+        background: linear-gradient(to bottom, #febbbb 0%, #fe9090 45%, #ff5c5c 100%);
+      }
+
+      .remove-button {
+        margin: 0 0 0 5px;
+        padding: 0;
+        border: none;
+        background: none;
+        cursor: pointer;
+        vertical-align: middle;
+        font: bold 16px Arial, sans-serif;
+        color: #585858;
+
+        &:active {
+          color: #ff0000;
+        }
+      }
+    }
+
+    .tag-list {
+      padding: 0.07em;
+    }
+
+    .input {
+      margin: 1px;
+      font: inherit;
+      font-size: 0.88rem;
+      border-radius: 0;
+    }
+  }
+}
+
+.keywords[disabled] {
+
+  .tags {
+    background-color: #eeeeee;
+
+    .tag-item {
+      background: #337ab7;
+      opacity: 0.65;
+    }
+
+    .input {
+      background-color: #eeeeee;
+    }
+  }
+}
+
+.input-group {
+
+  .keywords {
+    padding: 0;
+    display: table-cell;
+  }
+}
+
+.input-group .keywords:not(:first-child) .tags {
```

Comment:
@simonv3 I did do these ones! Refer commit: [Nested Elements](https://github.com/openfarmcc/OpenFarm/pull/829/commits/cbd799e8e5aeb68868ace016f97027709fd7c525)
Is Hound not updating?